interpreter/python/wrapper_decode.go

```diff
@@ -0,0 +1,26 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package python // import "go.opentelemetry.io/ebpf-profiler/interpreter/python"
+
+import (
+	"fmt"
+	"runtime"
+
+	"go.opentelemetry.io/ebpf-profiler/libpf"
+)
+
+func decodeStubArgumentWrapper(
+	code []byte,
+	codeAddress libpf.SymbolValue,
+	memoryBase libpf.SymbolValue,
+) (libpf.SymbolValue, error) {
```

Comment:
Seems this was like this, but why not inline this at the only call site, and use the `pfelf.File.Machine` to determine the machine type.